src/controller/auth.py

```diff
@@ -0,0 +1,8 @@
+from fastapi_sqlalchemy import db
+import requests
+
+
+def auth_account(id_token: str):
```

Comment:
```suggestion
def auth_by_token(id_token: str):
```
auth_account 보다는 토큰을 이용해서 인증한다는 이름을 붙여주는게 나을거 같네요